control/control.c

```diff
@@ -75,6 +76,8 @@ static void print_usage(const char *cmd)
 	printf("   push_guestos_config <guestos.conf> <guestos.sig> <guestos.pem>\n        (testing) Pushes the specified GuestOS config, signature, and certificate files.\n");
 	printf("   remove_guestos <guestos name>\n        Remove a GuestOS by the specified name. It will only remove the OS if no container is using it anymore.\n");
 	printf("   ca_register <ca.cert>\n        Registers a new certificate in trusted CA store for allowed GuestOS signatures.\n");
+	printf("   pull_csr <device.csr>\n        Pulles the device csr and stores it in <device.csr>.\n");
+	printf("   push_cert <device.cert>\n        Pushes back the device certificate provided by <device.cert>.\n");
```

Comment:
what is the command to push software signing certificates? push_cert might be a bit ambiguous.
our interface currently needs:
- pull device csr, push device cert
- push software signing trust anchor
- set/replace softtoken, resp. set softtoken password

anything else?

Reply:
ca_register see command above
softoken passwd change is not implemented yet and not part of this pull request